tools/IoTEdgeDevOps/DevOpsLibTest/IoTEdgeReleaseEnvironmentTest.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLibTest
+{
+    using System;
+    using DevOpsLib;
+    using DevOpsLib.VstsModels;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class IoTEdgeReleaseEnvironmentTest
+    {
+        [Test]
+        public void TestConstructorWithInvalidId([Values(-1)] int id)
+        {
+            ArgumentOutOfRangeException ex = Assert.Throws<ArgumentOutOfRangeException>(
+                () =>
+                    new IoTEdgeReleaseEnvironment(
+                        id,
+                        343406,
+                        VstsEnvironmentStatus.Succeeded));
+            Assert.True(ex.Message.StartsWith("Cannot be negative."));
+            Assert.AreEqual("id", ex.ParamName);
+        }
+
+        [Test]
+        public void TestConstructorWithInvalidDefinitionId([Values(-1, 0)] int definitionId)
```

Comment:
What about defintionId that is positive but not defined in the set that we care about? 
Do we lock those up as well?

Reply:
this is not what this class does.  This cannot be happened in ReleaseManagement as it use releasedefinitionid enum.